tests/components/geo_json_events/test_config_flow.py

```diff
@@ -0,0 +1,119 @@
+"""Define tests for the GeoJSON Events config flow."""
+from datetime import timedelta
+
+import pytest
+
+from homeassistant import config_entries, data_entry_flow
+from homeassistant.components.geo_json_events import DOMAIN
+from homeassistant.const import (
+    CONF_LATITUDE,
+    CONF_LOCATION,
+    CONF_LONGITUDE,
+    CONF_RADIUS,
+    CONF_SCAN_INTERVAL,
+    CONF_URL,
+)
+from homeassistant.core import HomeAssistant
+
+from tests.common import MockConfigEntry
+from tests.components.geo_json_events.conftest import URL
+
+pytestmark = pytest.mark.usefixtures("mock_setup_entry")
+
+
+async def test_duplicate_error_user(
+    hass: HomeAssistant, config_entry: MockConfigEntry
+) -> None:
+    """Test that errors are shown when duplicates are added."""
+    conf = {
+        CONF_URL: URL,
+        CONF_LOCATION: {CONF_LATITUDE: -41.2, CONF_LONGITUDE: 174.7, CONF_RADIUS: 25.0},
+    }
```

Comment:
I'm not sure having these as variable make it easier to follow the code.
I would just pass the data directly on line 41.

Or at the very least rename the variable to `user_input`

Please take this into consideration for all the other tests below.

Reply:
```suggestion
    user_input = {
        CONF_URL: URL,
        CONF_LOCATION: {CONF_LATITUDE: -41.2, CONF_LONGITUDE: 174.7, CONF_RADIUS: 25.0},
    }
```

Reply:
OK, I see what you mean. I decided to change all occurrences to pass the data directly.